test/rai/pipeline_adult_analyse.yaml

```diff
@@ -96,4 +96,4 @@ jobs:
       insight_1: ${{parent.jobs.causal_01.outputs.causal}}
       insight_2: ${{parent.jobs.counterfactual_01.outputs.counterfactual}}
       insight_3: ${{parent.jobs.error_analysis_01.outputs.error_analysis}}
```

Comment:
Can we update one existing example / add new test case using the new input?

Reply:
Adding a new test now!